leveldbjni-linux64-aarch64/pom.xml

```diff
@@ -64,7 +64,7 @@
       </plugin>
       <plugin>
         <groupId>org.fusesource.hawtjni</groupId>
-        <artifactId>maven-hawtjni-plugin</artifactId>
+        <artifactId>hawtjni-maven-plugin</artifactId>
```

Comment:
Just curious why the change to the `pom.xml` here?

Reply:
The artifactId was changed in the 1.16 version of hawtjni. Since I updated it from 1.11, this ID had to be changed.